src/ucp/core/ucp_mm.c

```diff
@@ -479,6 +474,8 @@ ucp_memh_alloc(ucp_context_h context, void *address, size_t length,
         goto err;
     }
 
+    ucs_assert(memh->map_count == 1);
```

Comment:
1. assertv
2. how does it work? seems ucp_mem_map increments map_count only after calling this function